spec/enumerable_spec.rb

```diff
@@ -0,0 +1,219 @@
+require './my_each.rb'
+
+describe Enumerable do
+  let(:array) { Array.new([1, 2, 3]) }
```

Comment:
[Optional]
`Array.new` actually no needed. just [1, 2, 3] will do it. Now seems to me like you are trying to create a copy of an array. That is not needed.